Sources/scipio/Options.swift

```diff
@@ -45,6 +45,11 @@ struct BuildOptionGroup: ParsableArguments {
     @Flag(name: [.customShort("f", allowingJoined: false), .long],
           help: "Whether overwrite generated frameworks or not")
     var overwrite: Bool = false
+
+    @Flag(name: [.customLong("--strip-dwarf-symbols")],
+          inversion: .prefixedNo,
+          help: "Whether to strip DWARF symbol from built binary or not")
+    var shouldStripDWARFSymbols: Bool = false
```

Comment:
Let's put this between `shouldEnableLibraryEvolution` and `shouldOnlyUseVersionsFromResolvedFile`.